code/modules/clothing/glasses/night.dm

```diff
@@ -42,8 +42,8 @@
 
 
 /obj/item/clothing/glasses/night/m56_goggles
```

Comment:
still inconsistent?

Reply:
Mmm, people almost never change typepaths like that because it necessitates a lot more code changes than simply changing the name/description.